### PR TITLE
Docs: Expand .aoMapIntensity description

### PR DESCRIPTION
**Pull request by donmccurdy**
Clarification of the typical range for `.aoMapIntensity`.

Related: 

- https://discourse.threejs.org/t/ambient-occlusion-wrong-with-khronos-neutral-tone-mapper/68192/5

/cc @WestLangley

### CODE REVIEW

File: docs/api/en/materials/MeshBasicMaterial.html [hunk 1]
```diff
@@ -78,8 +78,10 @@ <h3>[property:Texture aoMap]</h3>
 
 		<h3>[property:Float aoMapIntensity]</h3>
 		<p>
-			Intensity of the ambient occlusion effect. Default is `1`. Zero is no
-			occlusion effect.
+			Intensity of the ambient occlusion effect. Typical ranges are 0-1, where
```
Comment (WestLangley) [inline docs/api/en/materials/MeshBasicMaterial.html:81]: > Range is [0, 1].

//

(the only valid values.)

Comment (donmccurdy) [reply docs/api/en/materials/MeshBasicMaterial.html]: Changed, removed 'typically' phrasing. 👍🏻

**Review by Mugen87 — APPROVED** [review]

**Comment by pailhead** [comment]
I would have made a PR but i might be a bit out of touch with these later versions. This is a great starting point, but if the number represents something from 0-100% we should then probably clamp it.

**Comment by WestLangley** [comment]
> we should then probably clamp it

three.js does not validate user data. That is your app's responsibility.

**Comment by pailhead** [comment]
>three.js does not validate user data

I may be misinterpreting this pattern:

https://threejs.org/docs/#api/en/materials/MeshPhysicalMaterial.sheenRoughness
https://github.com/mrdoob/three.js/blob/216398f77b220f2fa2277a9406aa316ad97fce16/src/renderers/webgl/WebGLMaterials.js#L410
https://github.com/mrdoob/three.js/blob/216398f77b220f2fa2277a9406aa316ad97fce16/src/renderers/shaders/ShaderChunk/lights_physical_fragment.glsl.js#L117


- Like this improvement, the docs do mention that the value is expected to be in the range from 0 to 1.
- However, 0 doesn't actually seem to be ever seen by the shader. The smallest value seems to be `0.07`.
- Passing 123.4567 will result in the value of 1 in the shader.

I don't think that this is validation per se. Whatever the pattern may be, it could be applicable here.

**Pull request merged**